normandy/recipes/filters.py

```diff
@@ -353,6 +353,29 @@ def capabilities(self):
         return {"jexl.context.env.version", "jexl.transform.versionCompare"}
 
 
+class DateRangeFilter(BaseFilter):
+
+    type = "date_range"
+    not_before = serializers.DateTimeField()
+    not_after = serializers.DateTimeField()
+
+    def to_jexl(self):
+        not_before = self.initial_data["not_before"]
+        not_after = self.initial_data["not_after"]
+
+        return "&&".join(
+            [
+                f'(normandy.request_time>"{not_before}"|date)',
```

Comment:
This should probably be `>=` instead of `>`, so that ranges align exactly. This doesn't really matter because time uses such small increments, but for these kind of range filters I think we should be consistent.

Reply:
Sorry, I wasn't quite clear here before. I think that just the "left" side should be `>=`, the right side should stay as `<`. That makes it so that the ranges line up nicely.

Consider a simplified example with real numbers instead of dates: With `>=` and `<` the two ranges `[0, 5)` and `[5, 10)` add up to exactly the range `[0, 10)`, and the first doesn't contain any values the second has, and vice versa. With `>`/`<`, there is a small gap in the middle (5 falls into neither), and with `>=`/`<=` there is one that is duplicated (5 falls into both).

Again, this doesn't really matter for dates, but it matters for some other range filters we might do: It doesn't really matter if the last second in a range is included, but it definitely matters if the last version is included or not. I think it would be good to be consistent anywhere we have ranges in bucket filters.

Reply:
ok I see what you are saying. 👍